network/src/protocols/wire/handshake/v1/mod.rs

```diff
@@ -62,8 +67,28 @@ impl ProtocolId {
             ProtocolId::StateSyncDirectSend,
             ProtocolId::DiscoveryDirectSend,
             ProtocolId::HealthCheckerRpc,
+            ProtocolId::ConsensusRpcCbor,
+            ProtocolId::ConsensusDirectSendCbor,
         ]
     }
+
+    pub fn to_bytes<T: Serialize>(&self, value: &T) -> anyhow::Result<Vec<u8>> {
```

Comment:
Could you use a typed error instead of anyhow?

Reply:
different formats return different error types, we only use it for logging so I thought anyhow is fine